depends/README.md

```diff
@@ -1,5 +1,13 @@
 ### Usage
 
+## Requirements
+
+On debian you will need 
+
+   sudo apt-get install build-essential curl pkg-config libtool automake python3
```

Comment:
```suggestion
Before attempting to build these dependencies you should first follow the build instructions for your operating system, under the root folder:

* `doc/build-unix.md`
* `doc/build-osx.md`
* `doc/build-windows.md`

These contain up to date lists for each operating system.
```

I note `python3` isn't in the `build-unix.md` list, though. I'm currently experimenting with Debian 10 and it looks like it's in by default?

Reply:
Python is used to build a package, but not for `build-unix.md`, when the installation is done via a package manager.

I noticed that because I was using a Debian container, who do not contain python by default, I was using the same environment for Unix build and `depends` build.

Without python on `make` in `depends`, I get the following error inside a `xcb_proto` folder :
```
root@82f94871ef01:~/dogecoin/depends/work/build/x86_64-pc-linux-gnu/xcb_proto/1.10-6fe04c33d4c# ./configure                                                                                      
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... no
checking for mawk... mawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking for xmllint... no
configure: WARNING: xmllint not found; unable to validate against schema.
checking for a Python interpreter with version >= 2.5... none
configure: error: no suitable Python interpreter found
```

I would recommend to keep python, would be skipped on fully installed distros, and used by lightweight version without it. Idk if it should be considered as a core dependency, I would say no, but could depend on the situation (for example could master Makefile from Dogecoin root can build depends ?).

Reply:
About suggested changes, build guides are used to install core dependencies, as well as the `depends` folder.

From my pov, for a basic install, doesn't really make sense to redirect to `build-*.md` instructions, user would have to choose one of the two paths to install dependencies, and then finish with `autogen.sh/configure/make` in both case.

I would reject suggested changes.

Reply:
I mean... I'm not sure of the scenario where they'd want the `depends` folder except to build core, at which point surely they follow the build guide?

Reply:
So when is `depends` used ? CI & gitian build only ?

As you said, I'm wondering how/if `depends` would be used for a basic installation. Someone with no rights on package manager ? Someone with a specific architecture (arm ?) ? Someone who wants all dependencies for his installation, or some specific configuration ? For performance ?

Documentation will depend on usage

Reply:
I think putting the info to the os-specific instructions makes sense

Reply:
> So when is `depends` used ? CI & gitian build only ?
I have used depends for a build setup where i did not yet had a gitian setup; and i have used it for experimenting on Qt as there are plugins for Emojis which I wanna try out. I guess you only need it if you want to build from core.